src/components/pages/LockScreen.tsx

```diff
@@ -23,6 +44,10 @@ export const LockScreen = (): JSX.Element => {
           <MenuItemSwitch
             primary={t('Nav.primary_biometry')}
             icon={Fingerprint}
+            onClick={(): void =>
+              void handleChange(setHasBiometry, webviewIntent)
+            }
+            checked={hasBiometry}
```

Comment:
nit: got a bit confused by `hasBiometry` name. I expected it to contain the "capability" and not the fact the user enabled it. Maybe `shouldUnlockWithBiometry`?

Reply:
I'll recheck naming convention when more settings are added but you're right